src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -255,44 +252,38 @@ export const ProgressBar = (): JSX.Element => {
-                                    open={true}
+                                    classes={{
+                                        root: classes.snackbarRoot,
+                                        anchorOriginBottomCenter: classes.bottomCenter,
+                                    }}
+                                    open={item.open}
                                     autoHideDuration={item.progress === 100 ? 3000 : null}
                                     onClose={(e, reason): void => handleRequestClose(e, reason, item.id)}
+                                    TransitionProps={{ timeout: 300, onExited: (): void => handleExited(item.id) }}
                                 >
-                                    <SnackbarContent
-                                        classes={{ root: classes.SnackbarContent }}
-                                        action={
-                                            <>
-                                                <Button
-                                                    variant="outlined"
-                                                    className={classes.closeButtonContainer}
-                                                    onClick={(): void => removeListItem(item.id, item.status)}
-                                                >
-                                                    {item.progress === 100 ? 'View' : 'Cancel'}
-                                                </Button>
-                                            </>
-                                        }
-                                        message={
-                                            <div className={classes.messageContainer}>
-                                                <div className={classes.iconContainer}>
-                                                    <Description className={classes.icon} />
-                                                </div>
-                                                <div className={classes.messageTextContainer}>
-                                                    <Typography variant={'subtitle1'} color={'inherit'}>
-                                                        {item.name}
-                                                    </Typography>
-                                                    <Typography variant={'subtitle2'} className={classes.subTitle}>
-                                                        {item.status}
-                                                    </Typography>
-                                                </div>
-                                            </div>
-                                        }
-                                    />
+                                    <div>
+                                        <MuiThemeProvider theme={createTheme(PXBThemes.blueDark)}>
+                                            <InfoListItem
```

Comment:
You may want to add a box shadow for this

Reply:
Could you please elaborate more?

Reply:
```
<InfoListItem style={{boxShadow: theme.shadows[4]}}
```
@huayunh is 4 the right elevation for this?

Reply:
i vote for 6